tests/modules/suite2p_registration/test_suite2p_registration_suite2p_utils.py

```diff
@@ -147,3 +149,138 @@ def test_compute_reference(self):
         # These are integer arrays so they should be identical in value.
         self.assertTrue(np.all(np.equal(self.original_reference,
                                         result_reference)))
+
+    def test_optimize_motion_parameters(self):
+        """
+        """
+        # Set parameters for the pipeline.
+        suite2p_args = {'h5py': self.h5_file_loc,
+                        'h5py_key': 'input_frames',
+                        'maxregshift': 0.2,
+                        'smooth_sigma_min': 0.65,
+                        'smooth_sigma_max': 1.15,
+                        'smooth_sigma_steps': 2,
+                        'smooth_sigma_time_min': 0.0,
+                        'smooth_sigma_time_max': 1.0,
+                        'smooth_sigma_time_steps': 2}
+
+        def create_ave_image_mock(ref_image, suite2p_args, batch_size=500):
+            image = np.zeros((self.n_frames, self.n_frames), dtype=int)
```

Comment:
Shouldn't the shape for this be `self.frames.shape[1:]`? I think `self.n_frames` is the number of timesteps.

Reply:
(probably doesn't matter)

Reply:
It is but it's also the shape of the frame. The data shape is (100, 100, 100). I'll the xy shape in the setUp block to avoid later confusion.